### DISCUSSION TITLE
How to get timestamp value for Pre Var

### DISCUSSION THREAD

**Discussion by gjrivero** [usebruno/bruno#545] [Q&A] [answered]
Hi!
I'm trying to set Var named **timestamp** with this script:

const { moment } = require("moment");
req.setVar("timestamp", moment());

But, **moment** function is not recognized.

Some help is great!
Thanks

**Comment by helloanoop** [comment, marked as answer]
Try without the `{ }`

```
const moment = require("moment");
```
